api/ws/connector.js

```diff
@@ -95,17 +115,30 @@ function Websockets(conf, logger) {
             connection.sendUTF(JSON.stringify(initMessageObject));
             connection.on('close', function() {
                 me.logger.info("Websocket connection closed.");
-                me.reconnect();
+                setTimeout(function() {
+                    me.reconnect();
+                }, parseInt(me.retryTime));
             });
             connection.on('message', function(message) {
-                me.logger.info('Fired STATUS: ', message.utf8Data);
-                me.onMessage(message.utf8Data);
+                parseMessage(message.utf8Data, function(err, messageObject) {
+                    if(!err) {
+                        if(messageObject.code === 200) {
```

Comment:
define all error/success codes in separate file as constants. So you won't compare messageObject.code with magic numbers